### PR TITLE
Pipeline SOP and Cleanup Ops

### PR DESCRIPTION
**Pull request by victorskl**
- Added initial pipeline cleanup sql
- Update cleanup SQL: add FastqListRow cleanup
- Added pipeline SOP, automation doc and cleanup ops

### CODE REVIEW

File: docs/pipeline/bssh_sqs_event_replay.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      "body": "{\"gdsFolderPath\":\"/200612_A01052_0017_BH5LYWDSXY\",\"gdsVolumeName\":\"umccr-raw-sequence-data-dev\",\"reagentBarcode\":\"null\",\"v1pre3Id\":\"623627\",\"dateModified\":\"2021-04-12T04:29:51.4800606Z\",\"acl\":[\"wid:9c481003-f453-3ff2-bffa-ae153b1ee565\"],\"flowcellBarcode\":\"1ADD1CKMC\",\"sampleSheetName\":\"SampleSheet.csv\",\"apiUrl\":\"https://api.aps2.sh.basespace.illumina.com/v2/runs/r.AnPk0ox1OU-pACGtvSEQC2\",\"name\":\"200612_A01052_0017_BH5LYWDSXY\",\"id\":\"r.AnPk0ox1OU-pACGtvSEQC2\",\"instrumentRunId\":\"200612_A01052_0017_BH5LYWDSXY\",\"status\":\"PendingAnalysis\"}",
+      "attributes": {
+        "ApproximateReceiveCount": "1",
+        "SentTimestamp": "1589509372323",
+        "SenderId": "AROARFCPI2IGZA4XHGYYY:kiam-kiam",
+        "ApproximateFirstReceiveTimestamp": "1589509373008"
+      },
+      "messageAttributes": {
+        "subscription-urn": {
+          "stringValue": "urn:ilmn:igp:us-east-1:YXdzLXVzLXBsYXRmb3JtOjEwMDAwNTM3OjBiYTU5YWUxLWZkYWUtNDNiYS1hM2I1LTRkMzY3YTQzYWJkNQ:subscription:sub.889",
```
Comment (reisingerf) [inline docs/pipeline/bssh_sqs_event_replay.json:15]: This is our tenant ID, right?
I don't think exposing this is a security risk though.

File: docs/pipeline/cleanup/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Cleanup SOP
+
+### Steps
+
+- Must up-to-date understanding with Pipeline Data Model
+- General strategy: preform cleanup ops before staging rerun trigger
+- Using SQL scripts to determine 
+  - affected GDS paths to be deleted
+  - affected workflows to be marked them `Failed`
```
Comment (reisingerf) [inline docs/pipeline/cleanup/README.md:9]: mention that DB clean up (`FastqListRow`) may be needed?
(noted below, but perhaps worth to point out here in those "Steps". Notes are easily overlooked.)

File: docs/pipeline/misc-portal-lambda.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Misc Portal Lambda
+
+### Fastq lambda
```
Comment (reisingerf) [inline docs/pipeline/misc-portal-lambda.md:3]: The preferred way is to go through the API. 
Do you want to document this? Do we still have usage of this?

**Review by reisingerf — COMMENTED** [review]
Nice work!
Just a few minor comments.

File: docs/pipeline/bssh_sqs_event_replay.json (hunk 1 above)
Comment (victorskl) [reply docs/pipeline/bssh_sqs_event_replay.json:15]: Right. Oversight, I better rotate it.

File: docs/pipeline/cleanup/README.md (hunk 2 above)
Comment (victorskl) [reply docs/pipeline/cleanup/README.md:9]: It is a bit dynamic, but yah, I will pull it up.

File: docs/pipeline/misc-portal-lambda.md (hunk 3 above)
Comment (victorskl) [reply docs/pipeline/misc-portal-lambda.md:3]: Removing

**Comment by victorskl** [comment]
Flo; if this looks all good, pls help merge. I reckon, should be good!

**Review by reisingerf — APPROVED** [review]

**Pull request merged**